### PR TITLE
Add Basic Integration Test

### PR DESCRIPTION
**Pull request by cooperlees**
- Test a 2 package whitelisted bandersnatch run
- Assert the mirror run has correct output on the filesystem
- Also Run a verify - TODO: Inject bad files and ensure deleted (we don't test anything other than verify runs)
- Cleanup blacklist + whitelist logging to not be so spammy

All feedback welcome. Want to start basic and then add more edge cases.
Adresses #78

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,25 +6,28 @@ matrix:
...
       env: TOXENV=lint
+    - python: 3.7
+      env: TOXENV=py37
+    - python: 3.8-dev
+      env: TOXENV=py38
   allow_failures:
     - env: TOXENV=py38
 
 install:
+    - mkdir -v /tmp/pypi
```
[the remark this answers is not in the document]
Comment (cooperlees) [reply .travis.yml]: Sure. I didn't think it would hurt to have it on all test runs.
